src/index.ts

```diff
@@ -27,8 +27,8 @@ async function runMain() {
       if (process.platform === 'darwin') {
         MacBuilder.run(actionFolder, workspace, buildParameters);
       } else {
-        builtImage = await Docker.build({ path: actionFolder, dockerfile, baseImage });
-        await Docker.run(builtImage, { workspace, ...buildParameters });
+        //builtImage = await Docker.build({ path: actionFolder, dockerfile, baseImage });
+        await Docker.run(baseImage, { workspace, actionFolder, ...buildParameters });
```

Comment:
We have to either split between platforms or make Windows compatible with the same changes here.

Reply:
yeap,  I haven't touched windows yet,  I wanted feedback from you to see if this approach was good or not.